src/system/Form/Input.tsx

```diff
@@ -25,11 +24,16 @@ const inputStyles = {
 	variant: 'inputs.default',
 };
 
-const Input = React.forwardRef(
-	(
-		{ variant, label, forLabel, hasError = false, required, sx = {}, errorMessage, ...props },
-		ref
-	) => (
+interface InputProps extends ThemeInputProps {
+	label?: string;
+	hasError?: boolean;
+	required?: boolean;
+	forLabel?: string;
+	errorMessage?: string;
+	sx?: ThemeUIStyleObject;
```

Comment:
you can omit this, it's defined identically on `ThemeInputProps`

Reply:
changed in https://github.com/Automattic/vip-design-system/pull/251/commits/9618d7d8300376c3e6cd310f4cb53665ccbb5f4a